oracle/.env

```diff
@@ -7,8 +7,5 @@ CRONTAB = "*/10 * * * *"
 # bip39 mnenomic of oracle
 MNEMONIC = "equip town gesture square tomorrow volume nephew minute witness beef rich gadget actress egg sing secret pole winter alarm law today check violin uncover"
```

Comment:
Is this the real mnenomic? Let's add all .env files to .gitignore and create an .env.example file which contains something like `MNEMONIC = "secret words go here"`.